packages/react-composites/src/composites/common/icons.tsx

```diff
@@ -42,6 +44,8 @@ export const COMPOSITE_ONLY_ICONS = {
   LocalDeviceSettingsMic: <MicOn20Filled />,
   LocalDeviceSettingsSpeaker: <Speaker220Filled />,
   LocalPreviewPlaceholder: <VideoOff20Filled />,
+  /* @conditional-compile-remove-from(stable) */
```

Comment:
Add the feature name in the end like:
/* @conditional-compile-remove-from(stable): localVideoCameraSwithc */